### PR TITLE
Fix syslog-ng check

### PR DESCRIPTION
**Pull request by abenbachir**

### CODE REVIEW

**Comment by nidhanda** [comment]
Have you tested this on a LAD repro? Can we check if this works with a manual  ./diagnostic.py -enable command for LAD just to be sure?

**Comment by abenbachir** [comment]
> Have you tested this on a LAD repro? Can we check if this works with a manual ./diagnostic.py -enable command for LAD just to be sure?

I removed that SLES11 hack in LAD here https://github.com/Azure/azure-linux-extensions/pull/1104

File: test/code/plugins/oms_configuration_test.rb [hunk 1]
```diff
@@ -151,6 +151,7 @@ def test_load_configuration()
       assert_equal(TEST_UUID, Configuration.uuid, "VM UUID should be loaded")
       assert_not_equal(nil, Configuration.cert, "Certificate should be loaded")
       assert_not_equal(nil, Configuration.key, "Key should be loaded")
+      p "get_azure_region_from_imds=", OMS::Configuration.get_azure_region_from_imds()
```
Comment (nidhanda) [inline test/code/plugins/oms_configuration_test.rb:154]: Why are we doing a print here instead of assert check?

Comment (abenbachir) [reply test/code/plugins/oms_configuration_test.rb]: testing to fix the UT

**Pull request merged**